### PR TITLE
Added test scenario for Persons Form Card navigtion

### PR DESCRIPTION
**Pull request by mp-sgent**

### CODE REVIEW

**Comment by mp-sgent** [comment]
image

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/persons/EditPersonPage.java [hunk 1]
```diff
@@ -103,8 +103,16 @@
           "//*[contains(text(),'The case person was added as an event participant to the selected event.')]");
   public static final By SEE_EVENTS_FOR_PERSON =
       By.cssSelector("div#See\\ events\\ for\\ this\\ person");
+  public static final By SEE_CASES_FOR_PERSON = By.id("See cases for this person");
+  public static final By SEE_CONTACTS_FOR_PERSON = By.id("See contacts for this person");
+  public static final By EDIT_CASES = By.id("edit-case-0");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/persons/EditPersonPage.java:108]: please add also an identifier  _BUTTON _ICON etc, something to understand exactly which element is in UI

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/persons/EditPersonPage.java [hunk 2]
```diff
@@ -103,8 +103,16 @@
...
+  public static final By EDIT_CONTACTS = By.id("edit-contact-0");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/persons/EditPersonPage.java:109]: same as above

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/EditPersonSteps.java [hunk 3]
```diff
@@ -120,6 +122,38 @@ public EditPersonSteps(
...
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+    Then(
+        "I click on Edit Contact button from Contacts card on Edit Person page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EDIT_CONTACTS);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+    Then(
+        "I click on Edit Immunization button from Immunization card on Edit Person page",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/EditPersonSteps.java:151]: let's mention also that the action is performed for the immunization created via API, so we can differenciate it from other immunizations which can be created in UI flows in other steps/pages

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Persons.feature [hunk 4]
```diff
@@ -16,4 +16,37 @@ Feature: Edit Persons
     And While on Person edit page, I will edit all fields with new values
     And I edit all Person primary contact details and save
     Then I click on save button from Edit Person page
-    And I check that previous edited person is correctly displayed in Edit Person page
\ No newline at end of file
+    And I check that previous edited person is correctly displayed in Edit Person page
+
+    @issue=SORDEV-8469
+  Scenario: Form card navigation in Edit Person Directory
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/Persons.feature:22]: please rename it to reflect a relevant test

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Comment by mp-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/56/allure/

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**